src/ontology/mondo-ingest.Makefile

```diff
@@ -96,6 +96,9 @@ $(COMPONENTSDIR)/ncit.owl: $(TMPDIR)/ncit_relevant_signature.txt | component-dow
 		rename --mappings config/property-map.sssom.tsv --allow-missing-entities true --allow-duplicates true \
 		query --update ../sparql/rm_xref_by_prefix.ru \
 		remove -T $(TMPDIR)/ncit_relevant_signature.txt --select complement --select "classes individuals" --trim false \
+			--drop-axiom-annotations NCIT:P378 \
+			--drop-axiom-annotations NCIT:P383 \
+			--drop-axiom-annotations NCIT:P384 \
```

Comment:
I see the synonyms and I do these axioms on them. But I haven't fully wrapped my mind (yet) around how this robot remove command would be removing those synonyms or how this solution works.

Reply:
Sorry I didnt have time last night to neither make a proper PR nor explain it.

In the next step

```
		remove -T config/properties.txt --select complement --select properties --trim true \
```

removes all properties from NCIT which are not in `config/properties.txt`. Or I should say it more brutally:

It removes _all axioms in which a property exist that is not in this list_.

NCIT:P383, for example, is not in `config/properties.txt`.

Now here is the key (and the problem)

If you find this axiom:

```
id: MONDO:123
synonym: "a" [NCIT:P383="YO"]
```

ROBOT recognises NCIT:P383 part of the axiom.. and deletes it.

Reply:
Ahhh... I see. In RDF/XML, which I'm most familiar, it looks like the synonym is "just an annotation property", and that the axiom for NCIT:P383 is "a separate axiom on that property".

But I see that way of modeling it / thinking about it is not entirely correct, and that's not how robots sees it either.

Reply:
Interesting. I saw that nci had the synonyms represented in two ways in the class:
1) oio:hasExactSynonym as:`<oboInOwl:hasExactSynonym>Refractory Mucinous Carcinoma Peritonei</oboInOwl:hasExactSynonym>`
2) separate axioms on the class
```
<owl:Axiom>
        <owl:annotatedSource rdf:resource="http://purl.obolibrary.org/obo/NCIT_C179180"/>
        <owl:annotatedProperty rdf:resource="http://www.geneontology.org/formats/oboInOwl#hasExactSynonym"/>
        <owl:annotatedTarget>Refractory Mucinous Carcinoma Peritonei</owl:annotatedTarget>
        <obo:NCIT_P383>SY</obo:NCIT_P383>
        <obo:NCIT_P384>NCI</obo:NCIT_P384>
    </owl:Axiom>
```

Are you saying that ROBOT removes `NCIT_P383` and also then removes `hasExactSynonym` as a downstream effect?

Reply:
Exactly, and the reason for this is that the owl api will parse this RDFXML structure into a single owlaxiom instance with two axiom annotations!

Reply:
What trish copy/pasted is exactly what I observed as well.

Hmm, interesting. @matentzn Forgive my tangent, but as our feedback is also probably valued for ROBOT, and OWL API (which I guess we have a whole lot less sway over), do you think we are unique in being surprised by this behavior? Or do you think that what OWL API is doing here is poor design? Or are computational ontologies so complex that confusing cases like this are just inevitable?

Reply:
Joe - can you ask this in Slack? It's too much of a tangent for this PR. Thank you.